src/components/question.cjsx

```diff
@@ -57,50 +68,47 @@ module.exports = React.createClass
         removeAnswer={@removeAnswer}
         changeAnswer={@changeAnswer}/>)
 
-    <div>
+    <div className="question">
+
+      {if removeQuestion
+        <div>
+          {if canMoveRight
```

Comment:
Hm, yes this is a bit weird.  It's because they're all `float: right` so the order is reversed.  Let me see if we can remove some of the floats